source/lib/login.ts

```diff
@@ -78,15 +76,16 @@ export async function performLogin(
 	})
 
 	let responseUrl = new URL(loginResponse.url)
-	let responseMessage = responseUrl.searchParams.get('message')
+	// URLSearchParams.get requires a polyfill in react native
+	let responseMessage = responseUrl.href.includes('error=')
```

Comment:
I'm OK with this because it's a specific endpoint, and not a general method.

Reply:
(Plus, I suggested that Drew make this change)

Reply:
Alternative would be dragging in [`react-native-url-polyfill`](https://github.com/charpeni/react-native-url-polyfill/blob/main/js/URLSearchParams.js ) or doing what it does under the hood of importing `URLSearchParams` from 'whatwg-url-without-unicode.

I'm also okay with this approach for now.